react-app/src/components/Profile/UserForm/UserFormDate.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { UseFormSetValue } from 'react-hook-form';
+import { MuiPickersUtilsProvider, KeyboardDatePicker } from '@material-ui/pickers';
+// eslint-disable-next-line import/no-duplicates
+import DateFnsAdapter from '@date-io/date-fns';
+// eslint-disable-next-line import/no-duplicates
+import DateFnsUtils from '@date-io/date-fns';
+import { MaterialUiPickersDate } from '@material-ui/pickers/typings/date';
+
+interface Props {
+  setValue: UseFormSetValue<any>;
+  fieldName: string;
+  textName: string;
+  disabled: boolean;
+}
+
+const UserFormField: React.FC<Props> = ({ setValue, fieldName, textName, disabled }: Props) => {
```

Comment:
Change component name to match file name.

Reply:
done